python_apps/media-monitor/setup.py

```diff
@@ -0,0 +1,47 @@
+from setuptools import setup
+from subprocess import call
+import sys
+import os
+
+script_path = os.path.dirname(os.path.realpath(__file__))
+print script_path
+os.chdir(script_path)
+
+# Allows us to avoid installing the upstart init script when deploying on Airtime Pro:
+if '--no-init-script' in sys.argv:
+    data_files = []
+    sys.argv.remove('--no-init-script') # super hax
+else:
+    data_files = [('/etc/init', ['install/airtime-media-monitor.conf'])]
+    print data_files
+
+setup(name='airtime-media-monitor',
+      version='1.0',
+      description='Airtime Media Monitor',
+      url='http://github.com/sourcefabric/Airtime',
+      author='sourcefabric',
+      license='AGPLv3',
+      packages=['media-monitor', 'media-monitor2'],
+      scripts=['bin/airtime-media-monitor'],
+      install_requires=[
+          'amqplib',
+          'anyjson',
+          'argparse',
+          'configobj',
+          'docopt',
+          'kombu',
+          'mutagen',
+          'poster',
+          'PyDispatcher',
+          'pyinotify',
+          'pytz',
+          'wsgiref'
+      ],
+      zip_safe=False,
+      data_files=data_files)
```

Comment:
As mentioned earlier, we should create any needed log folders here, or media-monitor should be able to create /var/log/airtime/media-monitor itself. (I think the permissions are fixed now so that this is possible... ?)